README.md

```diff
@@ -2,162 +2,102 @@
-    ```sh
-    ssh -Nf -L 5432:localhost:5432 bety6.ncsa.illinois.edu
-    ```    
-3. Install a local instance of BETYdb
-    ```sh
-    load.bety.sh -c -u -m 99 -r 0 -w https://terraref.ncsa.illinois.edu/bety/dump/bety0/bety.tar.gz
-    load.bety.sh -m 99 -r 6 -w https://terraref.ncsa.illinois.edu/bety/dump/bety6/bety.tar.gz
-    ```
-
-#### Deploy Updates on Nebula / Kubernetes
+Note: If port 5432 is being used on your computer, you can use any other number. Just make sure to change the first number in the pairing to the number you choose.
 
-After updating the master branch on GitHub, a new Docker image is built on [Docker hub](https://hub.docker.com/r/terraref/traitvis-webapp/builds/). The following commands can be used to deploy and inspect this app as it is being served at traitvis.workbench.org
-
-Login to server (requires authentication)
+Initialize the database:
 
 ```sh
-ssh core@282.284.420.38 # = ip x 2
+docker-compose up -d postgres
+docker-compose run --rm bety initialize
+docker-compose run --rm bety sync
 ```
 
-See what is running in the terraref namespace, including extractors and traitvis
-
-```sh
-kubectl get pods --namespace=terraref
-``` 
+#### R packages
 
+This application requires the following R packages:
 
-The following restarts the container, which triggers the most recent image from [docker hub](https://hub.docker.com/r/terraref/traitvis-webapp/builds/) to be downloaded. 
+`shiny`, `shinythemes`, `scales`, `lubridate`, `dplyr`, `ggplot2`, `timevis`, `rgeos`, `leaflet`, `cronR`, `stringr`, `kableExtra`, `raster`, `mapview`, `leafem`, and `sf`.
 
-```sh
-kubectl delete pod terraref-traitvis-xxxx --namespace=terraref
-#xxxx is a random string.
-``` 
+#### Download image thumbnails
 
+Fullfield image thumbnails are available to be displayed under heatmaps. These thumbnails can be downloaded from the `/ua-mac/Level_2/rgb_fullfield/_thumbs/` endpoint on [Globus](https://www.globus.org/).
 
-Show status (i.e. ContainerCreating, Running, etc)
+These thumbs should be saved to the following path in your home directory `~/data/terraref/sites/ua-mac/Level_2/rgb_fullfield/_thumbs`.
 
-```sh
-kubectl get pods --namespace=terraref
-```
+#### How to run Shiny Application
 
-to see logs:
+Clone the [terraref/traitvis](https://github.com/terraref/traitvis-webapp) repository:
 
 ```sh
-kubectl logs -f terraref-traitvis-xxxx --namespace=terraref
+git clone https://github.com/terraref/traitvis-webapp.git
+cd traitvis-webapp
 ```
 
+The `cache-refresh.R` script needs to be run and requires a database connection. To connect to the local instance of BETYdb that you set up above, you will need to set the following connection parameters as environment variables.
 
-to get an interactive terminal 
+In your R console, run the following commands:
 
 ```sh
-kubectl --namespace=terraref exec -it terraref-traitvis-xxxx bash
+Sys.setenv(bety_host = 'localhost')
+Sys.setenv(bety_port = '5433')
```

Comment:
above when deploying docker this is set to 5432; perhaps it should be set to 5433 above?

Reply:
Yes it should. I have added a docker-compose.override.yml to the repo that exposes port 5433.